specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2024-01-01-preview/Recommendations.json

```diff
@@ -151,7 +151,6 @@
             "$ref": "#/parameters/RecommendationPatch"
           }
         ],
-        "x-ms-long-running-operation": true,
         "responses": {
           "202": {
```

Comment:
> "202": { [](http://example.com/codeflow?start=9&length=9)

202 is an invalid status code for operations that are not long running

Reply:
Same comment for other places doing the same thing

Reply:
Changed to 200.